benches/wasm-kernel/src/lib.rs

```diff
@@ -48,3 +40,84 @@ pub extern "C" fn bench_tiny_keccak(test_data: *const TinyKeccakTestData) {
 		keccak.finalize((*test_data).result);
 	}
 }
+
+pub struct RevComplementTestData {
+	input: ManuallyDrop<Box<[u8]>>,
```

Comment:
:+1: great, I really like this change

Reply:
yeah, a similar thing, but without unsafe : )